app/services/trainees/create_from_hesa.rb

```diff
@@ -168,8 +168,9 @@ def disabilities
     end
 
     def enqueue_background_jobs!
-      if FeatureService.enabled?(:integrate_with_dqt) && request_for_trn?
-        Dqt::RegisterForTrnJob.perform_later(trainee)
+      if FeatureService.enabled?(:integrate_with_dqt)
+        Trainees::Update.call(trainee: trainee)
```

Comment:
will this ordering cause lots of errors if they don't have their TRN yet?

Reply:
I'm not sure to be honest. @AbigailMcP what do you think?

Reply:
The `DQT::TraineeUpdate` service hits this endpoint: `/v2/teachers/update/#{trainee.trn}?birthDate=#{trainee.date_of_birth.iso8601}`

https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/services/dqt/trainee_update.rb#L21

so yes, we need to have the TRN back to be able to update them in DQT.